README.rst

```diff
@@ -5,8 +5,8 @@ Open Forms extension token exchange
 
 :Version: 0.1.0
 :Source: https://github.com/open-formulieren/open-forms-ext-token-exchange
-:Keywords: ``<keywords>``
-:PythonVersion: 3.9
+:Keywords: ``Open Forms Extension`` ``Keycloak token exchange``
```

Comment:
```suggestion
:Keywords: Open Forms Extension, Keycloak token exchange
```
I don't even know if there's a specific format to this, but the RST is not processed/index otherwise so it doesn't matter